__tests__/app.test.js

```diff
@@ -41,3 +41,42 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("status 200: should return an article object with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toBeInstanceOf(Object);
+        expect(article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          })
+        );
+      });
+  });
+  test("status:400, gives correct error message when given valid but nonexistent article id", () => {
```

Comment:
Your error status codes here are the wrong way around. 9999 is a valid id that may not have a user, so a 404 as it has not been found. fish is not a number and therefore invalid, a 400.